keyboards/dztech/dz60rgb/dz60rgb.c

```diff
@@ -0,0 +1,187 @@
+
+    {0, B_15,   A_15,   C_15},
+    {0, E_13,   D_13,   F_13},
+    {0, B_12,   A_12,   C_12},
+    {0, E_15,   D_15,   F_15},
+    {0, H_15,   G_15,   I_15},
+    {0, B_16,   A_16,   C_16},
+    {0, E_16,   D_16,   F_16},
+    {0, H_16,   G_16,   I_16},
+    {0, K_16,   J_16,   L_16},
+};
+
+const rgb_led g_rgb_leds[DRIVER_LED_TOTAL] = {
+    {{0|(13<<4)},   {17.23*13,  0}, 1},
+    {{0|(12<<4)},   {17.23*12,  0}, 1},
+    {{0|(11<<4)},   {17.23*11,  0}, 1},
+    {{0|(10<<4)},   {17.23*10,  0}, 1},
+    {{0|(9<<4)},    {17.23*9,  0}, 1},
+    {{0|(8<<4)},    {17.23*8,  0}, 1},
+    {{0|(7<<4)},    {17.23*7,  0}, 1},
+	{{0|(6<<4)},    { 17.23*6,  0}, 1},
+    {{0|(5<<4)},    { 17.23*5,  0}, 1},
+    {{0|(4<<4)},    { 17.23*4,  0}, 1},
+    {{0|(3<<4)},    { 17.23*3,  0}, 1},
+    {{0|(2<<4)},    { 17.23*2,  0}, 1},
+    {{0|(1<<4)},    { 17.23*1,  0}, 1},
+    {{0|(0<<4)},    { 17.23*0,  0}, 1},
+
+	{{1|(13<<4)},   {17.23*13,  16}, 1},
+    {{1|(12<<4)},   {17.23*12,  16}, 0},
+    {{1|(11<<4)},   {17.23*11,  16}, 0},
+    {{1|(10<<4)},   {17.23*10,  16}, 0},
+    {{1|(9<<4)},    {17.23*9,  16}, 0},
+    {{1|(8<<4)},    {17.23*8,  16}, 0},
+    {{1|(7<<4)},    {17.23*7,  16}, 0},
+	{{1|(6<<4)},    { 17.23*6,  16}, 0},
+    {{1|(5<<4)},    { 17.23*5,  16}, 0},
+    {{1|(4<<4)},    { 17.23*4,  16}, 0},
+    {{1|(3<<4)},    { 17.23*3,  16}, 0},
+    {{1|(2<<4)},    { 17.23*2,  16}, 0},
+    {{1|(1<<4)},    { 17.23*1,  16}, 0},
+    {{1|(0<<4)},    { 17.23*0,  16}, 1},
+
+	{{2|(13<<4)},   {17.23*13,  32}, 1},
+    {{2|(11<<4)},   {17.23*11,  32}, 0},
+    {{2|(10<<4)},   {17.23*10,  32}, 0},
+    {{2|(9<<4)},    {17.23*9,  32}, 0},
+    {{2|(8<<4)},    {17.23*8,  32}, 0},
+    {{2|(7<<4)},    {17.23*7,  32}, 0},
+	{{2|(6<<4)},    { 17.23*6,  32}, 0},
+    {{2|(5<<4)},    { 17.23*5,  32}, 0},
+    {{2|(4<<4)},    { 17.23*4,  32}, 0},
+    {{2|(3<<4)},    { 17.23*3,  32}, 0},
+    {{2|(2<<4)},    { 17.23*2,  32}, 0},
+    {{2|(1<<4)},    { 17.23*1,  32}, 0},
+    {{2|(0<<4)},    { 17.23*0,  32}, 1},
+
+	{{3|(13<<4)},   {17.23*13,  48}, 1},
+    {{3|(11<<4)},   {17.23*11,  48}, 0},
+    {{3|(10<<4)},   {17.23*10,  48}, 0},
+    {{3|(9<<4)},    {17.23*9,  48}, 0},
+    {{3|(8<<4)},    {17.23*8,  48}, 0},
+    {{3|(7<<4)},    {17.23*7,  48}, 0},
+	{{3|(6<<4)},    { 17.23*6,  48}, 0},
+    {{3|(5<<4)},    { 17.23*5,  48}, 0},
+    {{3|(4<<4)},    { 17.23*4,  48}, 0},
+    {{3|(3<<4)},    { 17.23*3,  48}, 0},
+    {{3|(2<<4)},    { 17.23*2,  48}, 0},
+    {{3|(1<<4)},    { 17.23*1,  48}, 0},
+    {{3|(0<<4)},    { 17.23*0,  48}, 1},
+
+	{{4|(13<<4)},   {17.23*13,  64}, 1},
+    {{4|(11<<4)},   {17.23*11,  64}, 1},
+    {{4|(10<<4)},   {17.23*10,  64}, 1},
+    {{4|(9<<4)},    {17.23*9,  64}, 1},
+    {{4|(8<<4)},    {17.23*8,  64}, 1},
+    {{4|(5<<4)},    { 17.23*5,  64}, 0},
+    {{4|(2<<4)},    { 17.23*2,  64}, 1},
+    {{4|(1<<4)},    { 17.23*1,  64}, 1},
+    {{4|(0<<4)},    { 17.23*0,  64}, 1},
+
+};
+
+void bootmagic_lite(void)
```

Comment:
This isn't necessary, and is actually redundant.

https://docs.qmk.fm/#/feature_bootmagic?id=bootmagic-lite